shared/chat/conversation/search/index.js

```diff
@@ -181,18 +181,18 @@ class ThreadSearchDesktop extends React.Component<SearchProps & Props> {
               {this.props.hasResults() && (
                 <Kb.Box2 direction="horizontal" gap="tiny">
                   <Kb.Text type="BodySmall" style={styles.results}>
-                    {this.props.status === 'done' && this.props.hits.length === 0
+                    {noResults
                       ? 'No results'
                       : `${this.props.selectedIndex + 1} of ${this.props.hits.length}`}
                   </Kb.Text>
                   <Kb.Icon
-                    color={Styles.globalColors.black_50}
-                    onClick={this.props.onUp}
+                    color={noResults ? Styles.globalColors.black_35 : Styles.globalColors.black_50}
+                    onClick={!noResults ? this.props.onUp : null}
```

Comment:
This feels a little weird to me, probably better done in the container.

Reply:
Good point.

Reply:
@mmaxim, I can't seem to get it to work. The goal is to remove the hover styling which Icon's `onClick` prop provides.

Reply:
Ah ok, no worries then.